`make test` fails with error using python 3.10

See below for the error. I think this is a problem with pytest 6.1.1 -- fixed in pytest 6.2.5. (See https://stackoverflow.com/questions/69528842/an-error-while-trying-to-execute-tests-on-python-3-10-with-pytest)


$ make test
PYTHONPATH=src python -m pytest
===================================== test session starts =====================================
platform darwin -- Python 3.10.6, pytest-6.1.2, py-1.11.0, pluggy-0.13.1
rootdir: /Users/jason/src/pyprojroot
collected 0 items / 1 error

=========================================== ERRORS ============================================
_____________________________ ERROR collecting tests/test_here.py _____________________________
../../.pyenv/versions/3.10.6/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1050: in _gcd_import
    ???
<frozen importlib._bootstrap>:1027: in _find_and_load
    ???
<frozen importlib._bootstrap>:1006: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:688: in _load_unlocked
    ???
venv/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:162: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
venv/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:359: in _rewrite_test
    co = compile(tree, fn_, "exec", dont_inherit=True)
E   TypeError: required field "lineno" missing from alias
=================================== short test summary info ===================================
ERROR tests/test_here.py - TypeError: required field "lineno" missing from alias
!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!
====================================== 1 error in 0.21s =======================================
make: *** [test] Error 2

--- a/Makefile
+++ b/Makefile
@@ -1,23 +1,23 @@
 .PHONY: deploy
 deploy:
 	rm -f dist/*
-	python setup.py sdist bdist_wheel
-	python -m twine upload dist/*
+	python3 setup.py sdist bdist_wheel
+	python3 -m twine upload dist/*
 
 .PHONY: test_install
 test_install:
-	python -m pip install --index-url https://pypi.org/simple/ --no-deps --upgrade pyprojroot
+	python3 -m pip install --index-url https://pypi.org/simple/ --no-deps --upgrade pyprojroot
 
 .PHONY: lint
 lint:
-	python -m mypy --strict src/pyprojroot
-	python -m flake8 src/pyprojroot tests
-	python -m black --check --diff src/pyprojroot tests
+	python3 -m mypy --strict src/pyprojroot
+	python3 -m flake8 src/pyprojroot tests
+	python3 -m black --check --diff src/pyprojroot tests
 
 .PHONY: fmt
 fmt:
-	python -m black src/pyprojroot tests
+	python3 -m black src/pyprojroot tests
 
 .PHONY: test
 test:
-	PYTHONPATH=src python -m pytest
+	PYTHONPATH=src python3 -m pytest

